src/context.rs

```diff
@@ -1085,6 +1085,11 @@ impl Context {
                 .unwrap_or_default(),
         );
 
+        res.insert(
+            "storage_usage",
+            format!("{:#?}", self.get_space_usage().await),
```

Comment:
```suggestion
           self.get_space_usage().await.log_err().unwrap_or_default()
```

Otherwise this will always format `Ok`.
See also
https://github.com/chatmail/core/blob/main/STYLE.md#errors

Reply:
Anyway, this is likely to be removed from the info as it is a multiline string with `=` signs and breaks existing parsing in python tests and probably elsewhere where it is expected that info is a bunch of lines with a single `=` in between